include/prepared_statement.h

```diff
@@ -0,0 +1,42 @@
+#include <inttypes.h>
+
+/* The format that our prepared statements are called on the Postgres server */
+#define PREPARED_STMT_NAME_FORMAT "PS_%" PRIu64
```

Comment:
What about something like "PGBOUNCER_%" to both reduce collisions with existing apps (who might deallocate it) and to make the origin more obvious for things like pg_prepared_statements.

Reply:
Just few notes to @JelteF reply:
- there is close command in libpq protocol which drops prepared statement. The problem was that libpq library had no function for sending it, which is fixed by [https://github.com/postgres/postgres/commit/28b5726]. But if a driver works with socket directly, not through libpq library, then it is not a problem to send this command.
- pgbouncer right now doesn't take in account prepared statement names at all: it looks just at query content itself. So if you prepare the same statement with two different names, then pgbouncer will use just one. As a consequence names of prepared statements given by pgbpouncer `PS_%` do not interleave with prepared statement name assigned by client. Certainly with the exception of statements prepared using SQL `PREPARE` command. In this case we can really get conflict. So may be using "PGBOUNCER_" prefix instead of "PS_"really makes sense.
-

Reply:
Done. I agree it makes sense to avoid collisions like that.